src/Sylius/Bundle/AdminBundle/Resources/views/PaymentMethod/Gateways/paymentGateways.html.twig

```diff
@@ -0,0 +1,6 @@
+{% for name, gateway in gatewayFactories %}
+    {% set createRouteName = metadata.applicationName~'_admin_'~metadata.name~'_create' %}
```

Comment:
`metadata.applicationName` is always `sylius`
`metadata.name` is alway `payment_method`

so we can hardcode the route name